packages/xo-web/src/xo-app/vm/tab-general.js

```diff
@@ -207,9 +211,13 @@ export default connectStore(() => {
       {isEmpty(vm.current_operations) ? null : (
         <Row className='text-xs-center'>
           <Col>
-            <h4>
-              {_('vmCurrentStatus')} {map(vm.current_operations)[0]}
-            </h4>
+            <h4>{_('vmCurrentStatus')}</h4>
+            {map(tasks, task => (
+              <p>
+                <strong>{task.name_label}:</strong>{' '}
+                <span>{Math.round(task.progress * 100)}%</span>
```

Comment:
I don't think we should display the progress if equals to 0, because:

- some actions don't have a progress, they are just pending and then they finish
- it's not a big deal to not display the progress only for the first percentile.